### PR TITLE
Topic 00 exercise 01

### PR DESCRIPTION
**Pull request by fyunes**
new branch name-
exercise 01- i, ii, iii, iv, v, vi.

### CODE REVIEW

File: Topic-00-Exercise-01/ex1.html [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+                        <li>Buy groceries</li>
+                        <li>Eath lunch</li>
+                        <li>Sleep Eternally</li>
+
+                  </ol>
+            </article>
+
+
+</body>
+
```
Comment (DemianMoschin-GLB) [inline Topic-00-Exercise-01/ex1.html]: extra line

File: Topic-00-Exercise-01/ex1.html [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+</html>
```
Comment (DemianMoschin-GLB) [inline Topic-00-Exercise-01/ex1.html]: add empty line after this

File: Topic-00-Exercise-01/iv.html [hunk 3]
```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+
```
Comment (DemianMoschin-GLB) [inline Topic-00-Exercise-01/iv.html]: Extra line

File: Topic-00-Exercise-01/iv.html [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Exercise 01 iv</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
```
Comment (DemianMoschin-GLB) [inline Topic-00-Exercise-01/iv.html]: extra line

File: Topic-00-Exercise-01/iv.html [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+                </tr>
+                <tr>
+                    <td>Other Expenses</td>
+                    <td>2016</td>
+                    <td>$435000</td>
+                </tr>
+            </table>
+
+        </article>
+
```
Comment (DemianMoschin-GLB) [inline Topic-00-Exercise-01/iv.html]: Remove extra lines

File: Topic-00-Exercise-01/iv.html [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+
+    </section>
+
```
Comment (DemianMoschin-GLB) [inline Topic-00-Exercise-01/iv.html]: Same as above

File: Topic-00-Exercise-01/iv.html [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+
+
+</body>
+
+</html>
```
Comment (DemianMoschin-GLB) [inline Topic-00-Exercise-01/iv.html]: Add empty line after

File: Topic-00-Exercise-01/v.html [hunk 8]
```diff
@@ -0,0 +1,86 @@
...
+            <select name="Sports" id="sp1" required>
+                <option value="">Choose:</option>
+                <option value="Football">Football</option>
+                <option value="Basktball">Basketball</option>
+                <option value="Swimming">Swimming</option>
+                <option value="Hockey">Hockey</option>
+            </select>
+            <br>
+
+
```
Comment (DemianMoschin-GLB) [inline Topic-00-Exercise-01/v.html]: Extra line

File: Topic-00-Exercise-01/v.html [hunk 9]
```diff
@@ -0,0 +1,86 @@
...
+            <textarea name="textarea" id="ta1" cols="30" rows="10" required></textarea>
+            <br>
+
+            <button type="submit" value="Submit">Submit</button>
+            <button type="reset" value="Reset">Reset</button>
+        </form>
+
```
Comment (DemianMoschin-GLB) [inline Topic-00-Exercise-01/v.html]: Same as above

File: Topic-00-Exercise-01/v.html [hunk 10]
```diff
@@ -0,0 +1,86 @@
...
+
+
+    </section>
+
+
```
Comment (DemianMoschin-GLB) [inline Topic-00-Exercise-01/v.html]: same as above

File: Topic-00-Exercise-01/v.html [hunk 11]
```diff
@@ -0,0 +1,86 @@
...
+</body>
+
```
Comment (DemianMoschin-GLB) [inline Topic-00-Exercise-01/v.html]: Same as above

File: Topic-00-Exercise-01/ex1.html (hunk 1 above)
Comment (fyunes) [reply Topic-00-Exercise-01/ex1.html]: fixed

File: Topic-00-Exercise-01/ex1.html (hunk 2 above)
Comment (fyunes) [reply Topic-00-Exercise-01/ex1.html]: fixed

File: Topic-00-Exercise-01/iv.html (hunk 3 above)
Comment (fyunes) [reply Topic-00-Exercise-01/iv.html]: fixed

File: Topic-00-Exercise-01/iv.html (hunk 4 above)
Comment (fyunes) [reply Topic-00-Exercise-01/iv.html]: fixed

File: Topic-00-Exercise-01/iv.html (hunk 5 above)
Comment (fyunes) [reply Topic-00-Exercise-01/iv.html]: fixed

File: Topic-00-Exercise-01/iv.html (hunk 7 above)
Comment (fyunes) [reply Topic-00-Exercise-01/iv.html]: fixed

Comment (fyunes) [reply Topic-00-Exercise-01/iv.html]: fixed

File: Topic-00-Exercise-01/v.html (hunk 8 above)
Comment (fyunes) [reply Topic-00-Exercise-01/v.html]: fixed

File: Topic-00-Exercise-01/iv.html (hunk 6 above)
Comment (fyunes) [reply Topic-00-Exercise-01/iv.html]: fixed

File: Topic-00-Exercise-01/v.html (hunk 9 above)
Comment (fyunes) [reply Topic-00-Exercise-01/v.html]: fixed

File: Topic-00-Exercise-01/v.html (hunk 10 above)
Comment (fyunes) [reply Topic-00-Exercise-01/v.html]: fixed

File: Topic-00-Exercise-01/v.html (hunk 11 above)
Comment (fyunes) [reply Topic-00-Exercise-01/v.html]: fixed

File: Topic-00-Exercise-01/v.html [hunk 12]
```diff
@@ -0,0 +1,86 @@
...
+            <textarea name="textarea" id="ta1" cols="30" rows="10" required></textarea>
+            <br>
+
+            <button type="submit" value="Submit">Submit</button>
+            <button type="reset" value="Reset">Reset</button>
+        </form>
+
+
+
```
Comment (DemianMoschin-GLB) [inline Topic-00-Exercise-01/v.html]: Still empty lines here.. please remove them  all

**Pull request merged**